backend/models/user.py

```diff
@@ -7,9 +7,21 @@ class User(UserMixin, db.Model):
     id = db.Column(db.Integer, primary_key=True)
     email = db.Column(db.String(64), index=True,
                       unique=True, nullable=False)
+    photo = db.Column(db.String(256), index=False,
+                      unique=False, nullable=False)
+    name = db.Column(db.String(64), index=False,
```

Comment:
I see here that the name is not an index and unique = false, but the migration file adds an index to the name column and marks it unique. Why is the migration not consistent with the code? Are all the migrations added in this PR?